README.md

```diff
@@ -5,7 +5,7 @@
 ## About
 
 This is the Klaw documentation repository. Welcome 👋 🎉
-You can find source code and all content for our ([klaw-project.io](https://www.klaw-project.io/) site here.
+You can find source code and all content for our ([Klaw-project.io](https://www.klaw-project.io/) site here.
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?